RAS18G3/everything_robot
# launch_files merge after wheel_odometry

This adjusts the er_wheels package to use parameters and namespaces to connect all topics together. Settings are found in the launch file.

## How to test
Connect the USB hub with the two motors and connect the USB hub with your computer / NUC. Then run
```console
roslaunch er_wheels wheel_odometry.launch
rviz
```
In rviz press add>Odometry and set topic to /wheel_odometry to see the robot moving when publishing via
```console
rostopic pub /cartesian_motor_controller/twist ...
```
